python/interpret_community/mimic/model_distill.py

```diff
@@ -20,6 +20,12 @@ def _soft_logit(values, clip_val=5):
     return np.clip(new_values, -clip_val, clip_val)
 
 
+def _inverse_soft_logit(values):
+    new_values = np.exp(values) / (1 + np.exp(values))
```

Comment:
could you use expit (inverse logit) function from scipy? https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.expit.html

Reply:
Done. Thanks for suggestion of the function.